src/main/kotlin/com/zepben/evolve/database/sqlite/readers/NetworkCIMReader.kt

```diff
@@ -305,6 +304,23 @@ class NetworkCIMReader(private val networkService: NetworkService) : BaseCIMRead
         return TownDetail(townName ?: "", stateOrProvince ?: "")
     }
 
+    private fun loadStreetDetail(table: TableStreetAddresses, resultSet: ResultSet): StreetDetail? {
```

Comment:
Can you please put the functions in each section in alphabetical order. It makes it much easier to spot if things have been missed later